client/opentracing/src/main/java/io/opentracing/impl/APMSpanBuilder.java

```diff
@@ -14,7 +14,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.opentracing;
+
+package io.opentracing.impl;
```

Comment:
Remind me again why we need to use their namespace, instead of `org.hawkular.apm.opentracing`, for instance